sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -386,6 +391,9 @@ static void _send_unicast(kernel_pid_t iface, uint8_t *dst_l2addr,
 
     DEBUG("ipv6: send unicast over interface %" PRIkernel_pid "\n", iface);
     /* and send to interface */
+#ifdef MODULE_NETSTATS_IPV6
+    gnrc_ipv6_netif_get_stats(iface)->tx_unicast_count++;
```

Comment:
What about keeping the netstats struct cached in a (static) global (to this unit file only) variable, assigned only once in `gnrc_ipv6_init`? You can then reuse that at several places in this file, instead of calling `gnrc_ipv6_netif_get_stats` each time.